tools/azsdk-cli/Azure.Sdk.Tools.Cli/.gitignore

```diff
@@ -0,0 +1,3 @@
+# generated by the ReadMeGeneratorTool, so you'll see it a lot...
+README.output.md
```

Comment:
I think this go in the repo root, the tool may be run from different directories in usage/test.

Reply:
Done.